default.nix

```diff
@@ -73,8 +73,8 @@ let
     else [x];
 
   dnas = with dnaPackages; [
-    happ-store
-    holo-hosting-app
+#    happ-store
+#    holo-hosting-app
```

Comment:
I can imagine this change being especially controversial. How do you feel about this? We're not using them currently right? It just saves me having to have my own copy of `default.nix` around all the time, and the risk of accidentally committing it etc.

Reply:
If we are clear that the `holo-hosting-app` zome calls that will be made by the `hp-admin` api will not go through and we can agree on it, I am **ok** with this change.
(At this stage of where we are at).

I do not know if you have already done this but I would also suggest you make a `Issue` on the `holo-nixpkgs` repo for the build issue that you are facing on a mac.

Reply:
@robbiecarlton - I second @joel on the fact that we should leave an issue on holo-nixpkgs with details about your machine and error log, so they can create a build that accommodates 86_64 Darwin machines (with Catalina).